# Message to request the matching of a cluster with a list of possible objects using texture descriptors
# cluster 
pr2_pick_perception/Cluster cluster

# list of possible objects
string[] object_ids 

# type of descriptor ORB, SIFT
string descriptor_type
